src/System.Private.Xml/src/System/Xml/Core/XmlTextWriter.cs

```diff
@@ -1337,15 +1360,16 @@ private void InternalWriteEndElement(bool longFormat)
             }
         }
 
+        private static readonly char[] s_closeTagEnd = new char[] { ' ', '/', '>' };
+
         private void WriteEndStartTag(bool empty)
         {
             _xmlEncoder.StartAttribute(false);
             for (int i = _nsTop; i > _stack[_top].prevNsTop; i--)
             {
                 if (!_nsStack[i].declared)
                 {
-                    _textWriter.Write(" xmlns");
-                    _textWriter.Write(':');
+                    _textWriter.Write(" xmlns:");
```

Comment:
I don't think it will have much effect as we're over the length threshold for individually copying each item. Write(string) uses String.CopyTo whereas Write(char[]) uses Buffer.BlockCopy.